app/src/main/res/values/strings.xml

```diff
@@ -1,5 +1,8 @@
 <resources>
     <string name="app_name">Health Gateway</string>
+    <string name="app_name_development">Health Gateway - DEV</string>
```

Comment:
If you prefix the Dev, Mock and STG,  it will be helpful pick that app from the device's apps menu. As the app name is long the suffix will get ellipsis.

Reply:
Thanks. Names have been updated